rackcity/models/it_model.py

```diff
@@ -19,7 +20,13 @@ class ITModel(models.Model):
         default='#394B59',
         validators=[validate_display_color],
     )
-    num_ethernet_ports = RCPositiveIntegerField(null=True, blank=True)
+
+    network_ports = ArrayField(
+        models.CharField(max_length=150, blank=True),
```

Comment:
per our discussion, I'd suggest we do not set `blank=True` here because we would actually set the "default" values ("1", "2", etc) a step further away from the model. we wouldn't want a row in the table to contain an array of empty strings since these would not be valid network port names

Reply:
this is the only change that requires a migration BTW